gamemodes/terrortown/gamemode/client/cl_main.lua

```diff
@@ -89,7 +89,8 @@ function GM:Initialize()
 
 	hook.Run("TTT2Initialize")
 
-	GAMEMODE.round_state = ROUND_WAIT
+	self.round_state = ROUND_WAIT
+	self.roundCount = 0
```

Comment:
Isn't there already some kind of counter, which is used for the mapchange in X rounds? could that be used here instead? i don't know how that one is implemented though, but i'd rather not have two counters/vars for essentially the same thing if we can avoid it

Reply:
Please give a look into my first comment here :)